src/Plugins/TokensTracker/Extensions.cs

```diff
@@ -28,7 +29,8 @@ public static bool NotNull(this StackItem item)
 
         public static string ToBase64(this ReadOnlySpan<byte> item)
         {
-            return item == null ? String.Empty : Convert.ToBase64String(item);
+            ReadOnlySpan<byte> nullData = null;
```

Comment:
?

Reply:
Because `item == null` will fail the format check in .net 9.

Reply:
We should add test to ensure if IsEmpty check null and not Length != 0

Reply:
Running dotnet format automatically changes `item == null` to `item.IsEmpty`. If you use `item.IsEmpty`, the unit test will fail, I tried it. So I've changed it to `ReadOnlySpan<byte> nullData = null; item == nullData;`

Reply:
just edit the `.editorconfig` and turn that off.